Multisig/UI/Onboarding/TermsView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  TermsView.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 28.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct TermsView: View {
+    @Binding var acceptedTerms: Bool
+    @Binding var isAgreeWithTermsPresented: Bool
+
+    @State private var showPrivacyPolicy = false
+    @State private var showTerms = false
+
+    var body: some View {
+        VStack(spacing: 12) {
+            BoldText("Please review our Terms of Use and Privacy Policy.")
+                .multilineTextAlignment(.center)
+                .fixedSize(horizontal: false, vertical: true)
+
+            VStack(alignment: .leading) {
+                HStack(alignment: .top) {
+                    Image("ico-bullet-point")
+                        .padding(.top, 8)
+                    Text("We do not collect demographic data such as age or gender.")
+                        .font(Font.gnoHeadline.weight(.medium))
+                        .fixedSize(horizontal: false, vertical: true)
+                }
+                HStack(alignment: .top) {
+                    Image("ico-bullet-point")
+                        .padding(.top, 8)
+                    Text("We collect anonymized app usage data and crash reports to ensure the quality of our app.")
+                        .font(Font.gnoHeadline.weight(.medium))
+                        .fixedSize(horizontal: false, vertical: true)
+                }
```

Comment:
the two things are copy-pasted, please extract it into a separate view

Reply:
good point. Ok